public/locales/ko/common.json

```diff
@@ -47,6 +47,7 @@
         "save": "정보 저장",
         "status_approved": "상태: 승인",
         "status_pending": "상태: 보류 중",
+        "status_expired": "TEMP HOLD",
```

Comment:
Don't add a placeholder. It's easier for third-party programs like https://www.electronjs.org/apps/i18n-manager to pick up if the translations are missing. 
```suggestion
```